server/util.go

```diff
@@ -242,6 +243,18 @@ func interceptConfigs(rootViper *viper.Viper) (*tmcfg.Config, error) {
 		return nil, fmt.Errorf("failed to merge configuration: %w", err)
 	}
 
+	// Adding ClientConfig
+	cliCfgFilePath := filepath.Join(configPath, "client.toml")
```

Comment:
I have to find an entry point where I can create "client.toml" and write it default client configuration into it. I hope this is a good spot for that, is not it?